### PR TITLE
[T3A2][T16-A3]Simaria Yesha Bharat

### PR DESCRIPTION
**Pull request by yeshasimaria**

### CODE REVIEW

**Comment by yeshasimaria** [comment]
Ready for review.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```
Comment (sooyj) [inline src/seedu/addressbook/commands/CommandResult.java:14]: Good work. Field changed to private.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -33,4 +33,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	public String getFeedbackToUser() {
```
Comment (sooyj) [inline src/seedu/addressbook/commands/CommandResult.java:36]: Correct! You now need a getter method for outsider.

File: src/seedu/addressbook/ui/TextUi.java [hunk 3]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if (resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
```
Comment (sooyj) [inline src/seedu/addressbook/ui/TextUi.java:128]: Correct! All outsider references need to go through the getter method to access information.

**Review by sooyj — COMMENTED** [review]
Good attempt. Field encapsulated, getter method added and all references updated.

**Pull request closed without merging**